### PR TITLE
Prepare for licensing change

### PR DESCRIPTION
**Pull request by dliappis**

### CODE REVIEW

File: cars/v1/defaults.ini [hunk 1]
```diff
@@ -6,4 +6,5 @@ type=car
 base=vanilla
```
Comment (danielmitterdorfer) [inline cars/v1/defaults.ini:6]: We also need to change all other configurations; some still reference the default distribution as a config base, e.g. https://github.com/elastic/rally-teams/blob/master/cars/v1/x-pack-security.ini.

**Review by danielmitterdorfer — COMMENTED** [review]
Thanks! I left a comment about the need to adapt other configs.

File: cars/v1/defaults.ini (hunk 1 above)
Comment (dliappis) [reply cars/v1/defaults.ini:6]: Oh right, good catch. Updated in https://github.com/elastic/rally-teams/pull/59/commits/0d068afcd302d000d94590a4fabcde184f8ac24c

**Review by danielmitterdorfer — APPROVED** [review]
Thanks! LGTM

**Comment by dliappis** [comment]
FYI @ebadyano and @danielmitterdorfer I've reverted back the basic-license (referencing vanilla) in https://github.com/elastic/rally-teams/pull/59/commits/3694bb64fceb5445e830324cbdb572bf377ed6bc and https://github.com/elastic/rally-teams/pull/59/commits/a4b52ebf9ebad286fce4808a8e67eba127d8b482 for backwards tooling compatibility.

**Review by danielmitterdorfer — APPROVED** [review]
Makes sense. I've re-approved your changes.

**Review by ebadyano — APPROVED** [review]
lgtm, thank you!

**Comment by dliappis** [comment]
Spot checks looked good, will merge this on Monday.

**Pull request merged**